### ISSUE TITLE
[Accessibility][Info&Relationship-Filter Dialog]: Ensures Logo image elements have alternate text or a role of none or presentation

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2139]
**User Experience** 
Users who rely on AT will face difficulty to locate the focus if the alt attribute is not defined for the image.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, run the Accessibility Insight tools to find the issue.

**Actual:** 
Bot Framework logo image doesn't have alternate text or a role of none or presentation.

**Expected:**
Ensures Bot Framework logo image have alternate text or a role of none or presentation.

**Note:**
This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Element path**
.css-isu7z6 > img

 **Snippet**
`<img src="/static/media/composerIcon.2e9dc5d0.svg" style="margin-left: 9px; margin-top: 6px;">`

**How to fix**
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute or the title attribute is empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"

**MAS Impacted:** [MAS1.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})

**Attachment:**
![Logo must have alt text](https://user-images.githubusercontent.com/60913131/75647293-8ffd7100-5c72-11ea-89ad-2c4fdf088012.png)

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
